src/dbnode/persist/fs/commitlog/types.go

```diff
@@ -107,7 +113,9 @@ type Series struct {
 	ID ident.ID
 
 	// Tags are the series tags
-	Tags ident.Tags // FOLLOWUP(prateek): wire Tags to commit log writer
+	Tags ident.Tags
+
+	TagIter ident.TagIterator
```

Comment:
Just curious, why do we have both now?  Is it an either/or argument?

Reply:
Can we create a wrapper type `TagsOrTagsIter` struct and perhaps a `type TagsType uint` enum like to specify which one to use?